Homework1

AlexanderKiselyov opened the pull request.

AlexanderKiselyov committed:
Delete index.html

AlexanderKiselyov committed:
Create index.html

AlexanderKiselyov committed:
Update index.html

Добавлены стили

AlexanderKiselyov committed:
Add style.css

Добавлены стили

aqus commented on index.html:
```diff
@@ -4,20 +4,29 @@
         <title>MyCard</title>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" type="text/css" href="styles/style.css">
```
что-то с отступами, будет выглядеть лучше, если будет на одном уровне с тегом `<meta>`

aqus commented on index.html:
```diff
@@ -4,20 +4,29 @@
-                <p>Student</p>
-                <button>Download CV</button>
-                <p>@2021 All rights reserved.</p>
+        <div id="wrapper">
```
Для стилизации лучше использовать `class`
Использовать id нужно в нескольких случаях, вот некоторые из них:
- ссылаться на объект через тег `<a>` (так называемый якорь)
- для связи с тегом `label` в формах

Это относится и к другим элементам этого документа.

aqus commented on index.html:
```diff
@@ -4,20 +4,29 @@
+        <div id="wrapper">
+            <div id="left-aside">
+                <div id="wrapper-left">
+					<img id="user-logo" src="https://cs11.pikabu.ru/post_img/2020/08/18/4/1597725632113587334.jpg"></img>
```
что-то снова отступы поехали
Плюс, для изображения нужно указывать дополнительный атрибут, который подсказывает, что лежит в картинке, если вдруг она не загрузится 
Это замечание также относится и к другим изображениям в этом документе

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,75 @@
+	justify-content: center;
+	margin: 5%;
+	height:70vh;
+	color: white;
```
color относится к наследуемому стилю, можно указать один раз в одном месте и не указывать несколько раз для разных элементов.

Не вижу, чтобы сбрасывались стили у body, он по умолчанию стилизуется браузером и скорее всего будет его стили будут мешать нам стилизовать по макету.

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,75 @@
+	height:70vh;
+	color: white;
+	background-color: grey;
+	border-radius: 50px;
```
Загляните в макет, кажется, что значение скругления там другое

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,75 @@
+#left-aside {
+	position: relative;
+	text-align: center;
+	width: 30vw;
```
предлагаю использовать абсолютные величины (пиксели), чтобы точнее следовать макету, ориентироваться на относительные величины, в данном случае на вьюпорт проблематично

aqus commented on styles/style.css:
```diff
@@ -0,0 +1,75 @@
+	align-self: center;
+	width: 200px;
+	height: 200px;
+	border-radius: 100px;
```
подсказка: чтобы сделать элемент круглым нужно задать border-radius: 50%

aqus commented:
Нужно проверить стилизации на предмет соответствия макету из фигмы, там есть все размеры.

aqus commented:
Еще момент, не вижу, чтобы сбрасывались стили у body
Предлагаю зайти на сайт ok.ru и поглядеть какие стили указаны на body, чтобы исправить стили учитывая наследование

AlexanderKiselyov committed:
Update style.css

AlexanderKiselyov committed:
Update index.html

Внесены исправления по файлам index.html и style.css. Добавлено оформление index.html по методологии БЭМ.
